docs/pages/web/buttonsocial.tsx

```diff
@@ -0,0 +1,118 @@
+import { ButtonSocial } from 'gestalt';
+import AccessibilitySection from '../../docs-components/AccessibilitySection';
+import CombinationNew from '../../docs-components/CombinationNew';
+import docGen, { type DocGen } from '../../docs-components/docgen';
+import GeneratedPropTable from '../../docs-components/GeneratedPropTable';
+import MainSection from '../../docs-components/MainSection';
+import Page from '../../docs-components/Page';
+import PageHeader from '../../docs-components/PageHeader';
+import QualityChecklist from '../../docs-components/QualityChecklist';
+import SandpackExample from '../../docs-components/SandpackExample';
+import main from '../../examples/buttonsocial/main';
+
+export default function DocsPage({ generatedDocGen }: { generatedDocGen: DocGen }) {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader description={generatedDocGen?.description} name={generatedDocGen?.displayName}>
+        <SandpackExample code={main} hideEditor name="Main ButtonSocial example" />
+      </PageHeader>
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description={`
+          - User login via social media or email
+        `}
+            title="When to use"
+            type="do"
+          />
+          <MainSection.Card
+            cardSize="md"
+            description={`
+            - If you need a button for a non-social media action.
+            - When a more generic button component would suffice.
+            - If the action does not involve user authentication or social media interaction.
+        `}
+            title="When not to use"
+            type="don't"
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Best practices" />
+      <AccessibilitySection name={generatedDocGen?.displayName} />
+
+      <MainSection name="Localization" />
```

Comment:
Nit, maybe follow up. Mind building a localized example. Check other examples to see how we do it.